404.html

```diff
@@ -59,6 +59,8 @@ <h1>Page Not Found</h1>
     1st mod to be ultimately DL to initializer-jokinjo from jokinjo/GH-Rep - after
     	(1) pushing to SoeJock/GH-Repo (2) Issuing pull-request to jokinjo/GH-Repo, and
     	(3) jokinjo/GH_Repo accepting pull-request 
+    2nd mod to test defaults of git push - to see if the defaults pertain to the current branch only,
+     or if the git push (w/no parameters) will pertain to all branches. 
```

Comment:
This experiment was a resounding success!  It showed that git push (no params) defaulted to pushing the local current branch to the current-branch-remote/current-branch-branch on the remote repo.